requirements/AOCS-GNC_AURORA_0001/taste_project/linux.dv.xml

```diff
@@ -0,0 +1,26 @@
+<DeploymentView>
+    <Node name="x86 Linux CPP_1" type="ocarina_processors_x86::x86.generic_linux" node_label="Node_1" namespace="ocarina_processors_x86">
```

Comment:
This DV seems to be incomplete. Where is MMI deployed?

Reply:
Removed
MMI uploaded